src/Umbraco.Web.UI.Client/src/common/services/editor.service.js

```diff
@@ -632,6 +632,7 @@ When building a custom infinite editor view you can use the same components as a
          */
         function iconPicker(editor) {
             editor.view = "views/common/infiniteeditors/iconpicker/iconpicker.html";
+            editor.size = "medium";
```

Comment:
I think it need to pass in the `size` property instead in case developers are implementing their own functionality use `iconPicker`. Could you e.g. add a button in a dashboard, which trigger iconPicker with size "large"?

Reply:
Yeah that's probably safer. Will give it a shot 👍🏻